readme-vars.yml

```diff
@@ -85,6 +85,7 @@ app_setup_block: |
   
 # changelog
 changelogs:
+  - { date: "19.06.20:", desc: "Compile wireguard tools and kernel module instead of using the ubuntu packages. Make module install optional. Imrpove verbosity in logs." }
```

Comment:
```suggestion
  - { date: "19.06.20:", desc: "Compile wireguard tools and kernel module instead of using the ubuntu packages. Make module install optional. Improve verbosity in logs." }
```

Reply:
Thanks, will fix that as well